src/main/java/com/google/sps/data/UserService.java

```diff
@@ -0,0 +1,54 @@
+package com.google.sps.data;
+
+import com.google.maps.GeocodingApi;
+import com.google.maps.model.AddressComponentType;
+import com.google.maps.model.AddressType;
+import com.google.maps.model.ComponentFilter;
+import com.google.maps.model.GeocodingResult;
+import com.google.maps.GeoApiContext;
+import com.google.maps.model.LatLng;
+import com.google.maps.model.LocationType;
+import java.util.Arrays;
+import java.util.logging.Logger;
+import io.github.cdimascio.dotenv.Dotenv;
+
+public class UserService {
+  
+  private Dotenv dotenv = Dotenv.configure().load();
```

Comment:
You left the dotenv in here which is fine but if we don't end up using it let's make sure to remove, do we already have an issue tracking our progress on the API keys?

Reply:
@AbakirH

Reply:
Per our conversation in standup, let's delete the Dotenv setup. @AbakirH will handle this separately.

Reply:
The dotenv setup is now deleted